scripts/docs/conf.py

```diff
@@ -131,6 +131,9 @@
 # myst reference config
 myst_heading_anchors = 5
 
+# Whether to create a Sphinx table of contents for the lists of class methods and attributes.
+numpydoc_class_members_toctree = False
```

Comment:
Why disable this?

Reply:
This clause is deleted now. It was needed because the `conf.py` has `numpydoc` extension which is not really needed for the ci since we are using reST style docstring not numpy style. I removed the `numpydoc` ext instead.